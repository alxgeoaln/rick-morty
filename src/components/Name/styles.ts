import { StyleSheet } from 'react-native'

const styles = StyleSheet.create({
    name: {
        fontSize: 30,
        fontWeight: "bold",
    }
})

export default styles;