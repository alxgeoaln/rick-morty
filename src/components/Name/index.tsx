import React from "react";
import { Text, TextProps } from 'react-native'

import styles from "./styles";

const Name: React.FC<TextProps> = ({ children, style }) => (
    <Text testID="card-name-test-id" style={[styles.name, style]}>{children}</Text>
)

export default Name;