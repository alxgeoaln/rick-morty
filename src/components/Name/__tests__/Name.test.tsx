import { render } from '@testing-library/react-native';
import Name from '..';

describe("Name component", () => {
    it("should match snapshot", () => {
        const snapshot = render(<Name />).toJSON()

        expect(snapshot).toMatchSnapshot();
    })
})