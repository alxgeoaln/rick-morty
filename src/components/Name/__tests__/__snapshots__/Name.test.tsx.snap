// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Name component should match snapshot 1`] = `
<Text
  style={
    [
      {
        "fontSize": 30,
        "fontWeight": "bold",
      },
      undefined,
    ]
  }
  testID="card-name-test-id"
/>
`;
